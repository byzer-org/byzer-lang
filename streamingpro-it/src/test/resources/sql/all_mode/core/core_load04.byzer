load csv.`/csv/skipNLines.csv` where skipFirstNLines="3" and skipLastNLines="4" and header="true" as skip_n_lines;
