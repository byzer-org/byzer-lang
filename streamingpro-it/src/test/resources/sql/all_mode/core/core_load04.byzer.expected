name,age,demo
jack,111,12312
jack2,123,12312
jack,135,12312
jack3,147,12312
jack,159,12312
jack4,171,12312
jack,183,12312
jack5,195,12312
jack,207,12312